cmd/bosun/conf/conf.go

```diff
@@ -470,6 +473,15 @@ func (c *Conf) loadGlobal(p *parse.PairNode) {
 			c.errorf("responseLimit must be > 0")
 		}
 		c.ResponseLimit = i
+	case "defaultRunEvery":
+		i, err := strconv.ParseInt(v, 10, 32)
```

Comment:
`strconv.Atoi` returns an int if that'd be easier, even though it's different. Whatever.